Makefile

```diff
@@ -56,6 +56,7 @@ LINGUAS=$(basename $(notdir $(wildcard po/*.po)))
 
 po/$(PACKAGE_NAME).js.pot:
 	xgettext --default-domain=$(PACKAGE_NAME) --output=$@ --language=C --keyword= \
+		--add-comments=Translators: \
```

Comment:
I don't see this in any other Cockpit project so that feels a bit strange. Should this be in all our projects?

Reply:
Yes.  I'll do a PR to starter-kit.